lib/integrations/externalauditstorage/bootstrap.go

```diff
@@ -214,6 +228,13 @@ func createBucket(ctx context.Context, clt BootstrapS3Client, bucketName string,
 		return trace.Wrap(awsutil.ConvertS3Error(err))
 	}
 
+	if _, err := clt.PutBucketTagging(ctx, &s3.PutBucketTaggingInput{
```

Comment:
```suggestion

	// s3:CreateBucket doesn't support tags, so the best we can do is
	// to tag the bucket shortly after creating it
	if _, err := clt.PutBucketTagging(ctx, &s3.PutBucketTaggingInput{
```

Reply:
Should we try to apply tags on the bucket at auth startup regardless? A Teleport run might successfully create the bucket and then fail to tag it, and then the bucket will forever stay tagless.

Reply:
If we can't tag, then the bootstrap fails and we abort the process.
We can discuss whether created resources should be cleaned up or not, but I'll leave that to another issue/PR. We already print which resources we are creating, so I think it's fine to just leave it as is.

As for trying to tag the bucket during Auth init. I don't think that's currently possible because we don't ask for the `s3:PutBucketTagging` policy when creating the Role's Inline Policy for the External Audit Storage.
https://github.com/gravitational/teleport/blob/8e7ff7efd3b261d41a70c4bf0fd527bf564adee8/lib/cloud/aws/policy_statements.go#L302-L318
We could ask for that extra permission, but that broadens our policy set, which has its own downsides.

Reply:
>If we can't tag, then the bootstrap fails and we abort the process.

Won't the next run see that the bucket exist and skip the tagging?

Reply:
If this fails, we have two options:
- either the user removes the buckets manually
- or the user removes the draft EAS configuration and starts over again, which will use a different bucket name.